.github/workflows/ufm_log_analyzer_ci_workflow.yml

```diff
@@ -2,23 +2,35 @@ name: Ufm log analyzer CI Workflow
 
 on: 
   push:
-     paths:
+    paths:
       - 'plugins/ufm_log_analyzer_plugin/**'
+      - '.github/workflows/ufm_log_analyzer_ci_workflow.yml'
+
 jobs:
   pylint:
     runs-on: ubuntu-latest
+
     steps:
     - name: Checkout code
       uses: actions/checkout@main
-    
+
     - name: Set up Python
       uses: actions/setup-python@main
       with:
-        python-version: 3.9  # Specify the Python version you want to use
-    
-    - name: Install dependencies
```

Comment:
why you decided to make it one step instead of two?

Reply:
It was always 2 steps, but the question is good.
No particular reason, but mainly since all plugins have this as one step

Reply:
let's make it 2, please, more explicit this way

Reply:
Tried to split, it will mean adding/copying some of the line.
I think it is ok to keep as it is, same step install the dependencies and making sure pytling sees them